tsconfig.json

```diff
@@ -8,10 +8,7 @@
     "noEmitOnError": true,
     "outDir": "temp",
     "sourceMap": true,
-    "baseUrl": "./node_modules",
-    "paths": {
-      "@ng-bootstrap/ng-bootstrap": ["../src/index"]
-    }
+    "baseUrl": "./node_modules"
```

Comment:
I'm not sure deleting `paths` here is a good thing. Maybe @pkozlowski-opensource can expound on that.